tests/integration/launchpad/test_anonymous_access.py

```diff
@@ -5,12 +5,27 @@
 from craft_application import launchpad
 
 
-def test_anonymous_login(tmp_path):
+@pytest.mark.parametrize(
+    "root",
+    [
+        pytest.param(
+            "staging",
+            marks=pytest.mark.xfail(
+                # Only xfail until the end of January.
+                datetime.date.today() < datetime.date(2025, 2, 1)
```

Comment:
err I meseed up with my suggestion

```suggestion
                datetime.date.today() < datetime.date(2025, 2, 1),
```

Reply:
heh

Reply:
And I committed it without testing locally!